packages/babel-helper-function-name/src/index.js

```diff
@@ -45,6 +45,20 @@ const visitor = {
   },
 };
 
+function getNameFromLiteralId(id) {
+  if (t.isNullLiteral(id)) {
+    return "null";
+  } else if (t.isRegExpLiteral(id)) {
```

Comment:
Since you are using the `return` statement, the `else if` statement becomes unnecessary. You could use `if` instead.

Reply:
That's true, i'll update